apps/src/dance/constants.js

```diff
@@ -39,3 +39,18 @@ export const DancelabReservedWords = [
 
 export const ASSET_BASE =
   'https://curriculum.code.org/images/sprites/dance_20191106/';
+
+export const oldDanceBackgroundEffects =
+  'circles, color_cycle, diamonds, disco_ball, fireworks, swirl, kaleidoscope, lasers, splatter, rainbow, snowflakes, galaxy, sparkles, spiral, disco, stars';
+
+export const newDanceBackgroundEffects =
```

Comment:
NIT: new and old feels a little odd here. For example, if we add more next year, do we name them "newerDanceBackgroundEffects"

Notes from our brainstorm: Perhaps "commonDanceBackgroundEffects" and "aiModuleDanceBackgroundEffects" or perhaps "v2ModuleDanceBackgroundEffects"

Reply:
Maybe even "danceBackgroundEffectsExtras"